### PR TITLE
Adjust the way preferences are accessed in the backend

### PR DESCRIPTION
**Pull request by jeremycline**
Previously it had access to the model directly, but when the preference
loading got unified it now has a dictionary.

Signed-off-by: Jeremy Cline <jeremy@jcline.org>

### CODE REVIEW

File: fmn/consumer/backend.py [hunk 1]
```diff
@@ -206,11 +206,11 @@ def read(queue_object):
     for recipient in recipients:
         user = recipient['user']
         t = time.time()
-        pref = PREFS.get('%s__%s' % (user, context))
+        pref = PREFS.get('%s_%s' % (user, context))
         print("pref retrieved in: %0.2fs" % (time.time() - t))
 
         try:
-            if not pref.should_batch:
+            if pref['batch_delta'] is None and pref['batch_count'] is None:
```
Comment (pypingou) [inline fmn/consumer/backend.py]: Should we use .get() here?

File: fmn/consumer/backend.py [hunk 2]
```diff
@@ -206,11 +206,11 @@ def read(queue_object):
     for recipient in recipients:
         user = recipient['user']
         t = time.time()
-        pref = PREFS.get('%s__%s' % (user, context))
+        pref = PREFS.get('%s_%s' % (user, context))
```
Comment (pypingou) [inline fmn/consumer/backend.py:209]: Did we fix the place where this is set?

Comment (jeremycline) [reply fmn/consumer/backend.py:209]: This was a silly thing I managed to somehow typo (obviously there's no test coverage) when converting stuff. The dictionary uses the key format fixed here.

File: fmn/consumer/backend.py (hunk 1 above)
Comment (jeremycline) [reply fmn/consumer/backend.py]: I don't think it should ever be missing, but it wouldn't hurt. Good call.

**Comment by pypingou** [comment]
👍 for me

**Pull request merged**